.travis.yml

```diff
@@ -36,6 +36,9 @@ before_script:
 
 script: "script/run_build 2>&1"
 
+env:
+  - USE_GIT_REPOS=1
+
```

Comment:
We should be automatically using git repos because theres no local ones

Reply:
That's strange because on failures, bundler was pointing to a local gem sources. With this ENV it now pass.